zhmcclient/_lpar.py

```diff
@@ -956,6 +1091,96 @@ def send_os_command(self, os_command_text, is_priority=False):
         self.manager.session.post(
             self.uri + '/operations/send-os-cmd', body)
 
+    @logged_api_call
+    def psw_restart(self, wait_for_completion=True, operation_timeout=None,
+                    status_timeout=None, allow_status_exceptions=False):
+        """
+        Restart the first available processor of this LPAR,
+        using the HMC operation "PSW Restart".
```

Comment:
Talking about the "first available processor" is confusing, IMO. The effect is that the LPAR is restarted. Can we use the description of `Partition.psw_restart()`, also for consistency?

Reply:
DONE - changed to "Restart this LPAR, using the HMC operation "PSW Restart"."